MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -9,6 +9,8 @@ protocol FeedViewControllerDelegate {
 }
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+	@IBOutlet var errorView: ErrorView?
```

Comment:
This property can be public so the tests can access the getter, and the setter can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet public private(set) var errorView: ErrorView?
```